src/bundle/Resources/public/scss/mixins/_drag-and-drop.scss

```diff
@@ -171,15 +171,18 @@
     }
 
     &--unavailable {
```

Comment:
Note: I haven't found any place where this modifier (BEM) was used and its styling seemed to be incorrect.